osquery/tables/system/darwin/user_groups.mm

```diff
@@ -55,6 +56,60 @@ void genODEntries(ODRecordType type, std::set<std::string> &names) {
   }
 }
 
+void genPolicyColumns(int uid, boost::property_tree::ptree& tree) {
+  @autoreleasepool {
+    ODSession* s = [ODSession defaultSession];
+    NSError* err = nullptr;
+    ODNode* root = [ODNode nodeWithSession:s name:@"/Local/Default" error:&err];
+    if (err != nullptr) {
+      TLOG << "Error with OpenDirectory node: "
+           << std::string([[err localizedDescription] UTF8String]);
```

Comment:
Is it okay to continue with this function after an error occurs? (Similar concerns below)

Reply:
Resolved, not sure why it's not going away on the PR.